parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -11,6 +11,9 @@ APMZ_DOWNLOADS_DIR="/opt/apmz/downloads"
 BPFTRACE_DOWNLOADS_DIR="/opt/bpftrace/downloads"
 UBUNTU_RELEASE=$(lsb_release -r -s)
 UBUNTU_CODENAME=$(lsb_release -c -s)
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
+    UBUNTU_VERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
```

Comment:
how is this different from `UBUNTU_RELEASE`?

Reply:
Like, not in any way:

```
azureuser@k8s-master-59690240-0:~$ lsb_release -r -s
18.04
azureuser@k8s-master-59690240-0:~$ grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "="
18.04
```

Reply:
fixed, thanks!